api/src/main/java/org/openmrs/module/eptsreports/reporting/data/converter/ObjectCounterConverter.java

```diff
@@ -0,0 +1,25 @@
+package org.openmrs.module.eptsreports.reporting.data.converter;
+
+import org.openmrs.module.reporting.data.converter.DataConverter;
+
+public class ObjectCounterConverter implements DataConverter {
```

Comment:
This index works, yes, but you are applying it to patients who have a given patient identifier of given type. What happens  if some patients will lack that identifier? it will throw null pointer. The safer way will be to apply it to patient/person ids, that way we ware sure such object will always be available

Reply:
Agreed and applied person id to the index